client/src/components/JurisdictionAdmin/BatchRoundDataEntry.tsx

```diff
@@ -1,606 +1,644 @@
+    }
+
+    removeSheet(index)
   }
 
+  const saveResults = (e: BaseSyntheticEvent) => {
+    cancelSheetRename()
+    setIsEditing(false)
+    handleSubmit(onValidSubmit, onInvalidSubmit)(e)
+  }
+
+  // ---------- Handlers (END) ----------
+
   return (
-    <div>
-      <div>
-        <p className={Classes.TEXT_LARGE}>
-          For each batch, enter the number of votes tallied for each
-          candidate/choice.
-        </p>
-        {resultsFinalizedAt && (
-          <Callout
-            icon="tick-circle"
-            intent="success"
-            style={{ margin: '20px 0 20px 0' }}
+    <ListAndDetail>
+      <List
+        search={{
+          placeholder: 'Search batches...',
+          setQuery: setSearchQuery,
+        }}
+      >
+        {filteredBatches.map(batch => (
+          <ListItem
+            key={batch.id}
+            onClick={() => selectBatch(batch.id)}
+            selected={batch.id === selectedBatchId}
           >
-            Results finalized
-          </Callout>
+            {batch.name}
+          </ListItem>
+        ))}
+      </List>
+
+      <Detail>
+        {!selectedBatch ? (
+          <p>Select a batch to enter audit results.</p>
+        ) : (
+          <>
+            <H4>{selectedBatch.name}</H4>
+            <Tabs
+              id={selectedBatch.name}
+              onChange={(newTabId: string) => {
+                cancelSheetRename()
+                setSelectedTabId(newTabId)
+              }}
+              // Defaults to false but noting explicitly for clarity. Rendering all tabs at all
+              // times is important for react-hook-form's state management
+              renderActiveTabPanelOnly={false}
+              selectedTabId={selectedTabId}
+            >
+              <Tab
+                id={VOTE_TOTALS_TAB_ID}
+                key={VOTE_TOTALS_TAB_ID}
+                panel={
+                  <BatchResultTallySheet
+                    areResultsFinalized={areResultsFinalized}
+                    cancelSheetRename={cancelSheetRename}
+                    confirmSheetRename={confirmSheetRename}
+                    contest={contest}
+                    deleteSheet={() => deleteSheet(0)}
+                    enableEditing={enableEditing}
+                    formMethods={formMethods}
+                    index={0}
+                    initiateSheetRename={() => initiateSheetRename(0)}
+                    isEditing={isEditing}
+                    isRenamingSheet={isRenamingSheet}
+                    isTotalsSheet={sheets.length > 1}
+                    savedResults={
+                      selectedBatch.resultTallySheets[0]?.results || {}
+                    }
+                    saveResults={saveResults}
+                    sheets={sheets}
+                  />
+                }
+              >
+                Vote Totals
+              </Tab>
+              {sheets.length > 1 &&
+                sheets.map((sheet, i) => (
```

Comment:
Once edge case that I'm still thinking through is the case that the number of sheets is so large that the tab bar overflows - looking into scrolling and wrapping

Reply:
Yes, I was thinking that could be an issue, though likely a rare case. My gut says scrolling will be better than wrapping.

Reply:
I've set up the most rudimentary horizontal scrolling when the number of sheets is large but have taken note in https://github.com/votingworks/arlo/issues/1655 (a catch-all issue for follow-ups) that there's room for improvement, e.g. horizontal scrolling of just the tab bar.

<img width="975" alt="jm" src="https://user-images.githubusercontent.com/12616928/198326349-6f0a5a16-397d-4ff7-a0ab-9923a84bad2e.png">

<img width="985" alt="te" src="https://user-images.githubusercontent.com/12616928/198326354-97c4dd39-7bc3-4d54-898c-588d66f7b7b3.png">